backend/FwLite/LcmCrdt/MediaServer/LcmMediaService.cs

```diff
@@ -50,25 +129,44 @@ public async Task DeleteResource(Guid fileId)
     /// <exception cref="FileNotFoundException"></exception>
     public async Task<ReadFileResponse> GetFileStream(Guid fileId)
     {
-        var localResource = await resourceService.GetLocalResource(fileId);
+        var localResource = await DownloadResourceIfNeeded(fileId);
```

Comment:
I don't like that we're not calling `DownloadResourceIfNeeded` 3 different times, if we want to implement retrying then that should go somewhere else, not in this method as it makes it much more messy and hard to follow.